api/identity.go

```diff
@@ -80,35 +79,6 @@ func (h IdentityHandler) List(ctx *gin.Context) {
 	ctx.JSON(http.StatusOK, resources)
 }
 
-// ListByApplication  godoc
```

Comment:
Just moved to later in the file.
Following a pattern of grouping all _sub-resource_ endpoints after all the _main_ endpoints.